example/cuplaReduce/src/main.cpp

```diff
@@ -0,0 +1,29 @@
+#include <vector>
+#include <numeric>
+
+#include <iostream>
+#include "cuplaVikReduce.hpp"
+
+int main()
+{
+    constexpr int size = 10;
+
+    std::vector<int> input(size);
+    std::iota(input.begin(), input.end(), 0);
+
+    int expected_sum = std::accumulate(input.begin(), input.end(), 0);
```

Comment:
Maybe use `std::reduce` here to showcase the differences? Or are we still bound to C++14?